flake.nix

```diff
@@ -31,8 +28,11 @@
     relayer-src.flake = false;
 
     # Chain Sources
+    gaia6-ordered-src.flake = false;
+    gaia6-ordered-src.url = github:informalsystems/gaia/v6.0.1-ordered;
+
     gaia6-src.flake = false;
-    gaia6-src.url = github:cosmos/gaia/v6.0.1;
+    gaia6-src.url = github:cosmos/gaia/v6.0.3;
```

Comment:
Gaia v6.0.3 is broken again with it using an older version of ibc-go: https://github.com/cosmos/gaia/issues/1284. So We have to revert to using v6.0.2 for now.

Btw next time you can push your branches to this repository directly so that I can also help push new changes.

Reply:
right, but there is a separate attr for the informal ordered gaia https://github.com/informalsystems/cosmos.nix/pull/64/files#diff-206b9ce276ab5971a2489d75eb1b12999d4bf3843b7988cbe8d687cfde61dea0R31-R32, and 6.0.2 has a bug in it that is causing chains to error, I know no one is using this, but just in case I want them to get the correct version that the community is using for now. https://discord.com/channels/669268347736686612/798937713474142229/944247197208289290

Reply:
Let's just remove `gaia6-ordered` then. It is a test fork of gaia that isn't meant to be merged. I can create a separate branch for testing using that.

According to https://github.com/cosmos/gaia/issues/1280, v6.0.3 is essentially a hotfix to revert to v6.0.0 as there are other bugs in v6.0.1 and v6.0.2. However v6.0.0 is broken and fail in one of ibc-rs integration tests. I think either we stick with v6.0.2 on master, or we'll have v6.0.3 on master and have branching strategies to place v6.0.2 in a branch like `next` or something.

Reply:
Actually a simpler solution is to just expose all minor versions, i.e. `gaia6_0_0` to `gaia6_0_3`. We just have to have `gaia6` to point to `gaia6_0_3` now, and ibc-rs can still use `gaia6_0_2` explicitly in its CI.